### DISCUSSION TITLE
Realtime.plot_summary question

### DISCUSSION THREAD

**Discussion by AP314159** [tropycal/tropycal#199] [Q&A] [answered]
I'm trying to add the sea surface temperature to a few of the charts I'm outputting. However, I cant seem to find the api reference for this.

Thanks for any help!

**Comment by tomerburg** [comment, marked as answer]
If you are referring to the maps on my website, those are made with additional code that is not a part of Tropycal. Right now that code is hard-coded specifically for my website, but once I have time to make it more generically applicable I'll add it as example code of various applications that Tropycal can be used for.

**Comment by RuthPetrie** [comment]
Hi, I am new to tropycal and I would like to customize some of the colours in the plots. Currently I have done this via a source install and hot fix to the code, is that the best way? Many thanks.

**Reply by tomerburg** [reply]
The answer depends on what plot(s) you are trying to customize and how you are trying to customize them. Unless absolutely necessary, it's generally not the best way to customize colors by editing the source code -- most functions provide options for customizing colors, or they can be customized from the axes instance returned from the function, though more customization functionality can be added to Tropycal if what you're trying to accomplish isn't possible by these methods I mentioned. Can you give an example plot you are trying to modify and what colors you want to change?

**Reply by RuthPetrie** [reply]
I was trying to go from this: 
![image](https://github.com/tropycal/tropycal/assets/15191131/aae597db-3347-46da-8c87-3157eb122a97)
to: 
![image](https://github.com/tropycal/tropycal/assets/15191131/666c88b6-965a-472e-8cd9-2981dd0d73b7)

I couldn't seem to make the changes I needed without directly editing the source. Many thanks for your help.
